packages/subgraph/tests/helpers/method-classes.ts

```diff
@@ -276,12 +273,11 @@ class ERC721ContractMethods extends ERC721Contract {
       throw new Error('Symbol is null');
     }
     // we cast to string only for stoping rust compiler complaints.
```

Comment:
Is this comment still needed? 
`createERC20TokenCalls` receives string arguments.